app/bundles/CoreBundle/Config/config.php

```diff
@@ -903,6 +903,8 @@
         'tmp_path'                        => '%kernel.project_dir%/var/tmp',
         'theme'                           => 'blank',
         'theme_import_allowed_extensions' => ['json', 'twig', 'css', 'js', 'htm', 'html', 'txt', 'jpg', 'jpeg', 'png', 'gif'],
+        'company_name'                    => 'Mautic',
```

Comment:
I'm wondering whether "Mautic" is the best default value here. Users may not understand this can be easily changed in the config. Can we change it to something like "Your Company (configurable)"?